packages/app/src/__deprecated__/renderer/wrappers/base-app.component.tsx

```diff
@@ -137,14 +137,6 @@ const BaseApp: FunctionComponent<Props> = ({
 }
 
 const isDeviceRestarting = (state: RootState & ReduxRootState): boolean => {
```

Comment:
Does the problem occur with backup and restore? Do we have an issue with this in our backlog?

Reply:
no, and no :)

Reply:
it's strange that we have a problem with update but not with other processes 🤔

Reply:
Thank you for checking it out <3